Nodsoft.WowsReplaysUnpack.Generators/SourceGenerationHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Nodsoft.WowsReplaysUnpack.Generators;
+
+public static class SourceGenerationHelper
+{
+	public const string Header = @"//------------------------------------------------------------------------------
```

Comment:
I highly recommend using raw strings instead of using verbatim strings.
Raw strings tend to be a lot more readable for longer text blocks